src/mudlet.cpp

```diff
@@ -122,7 +122,11 @@ const int mudlet::scmQDataStreamFormat_5_12 = 18;
 QPointer<TConsole> mudlet::mpDebugConsole = nullptr;
 QPointer<QMainWindow> mudlet::mpDebugArea = nullptr;
 bool mudlet::debugMode = false;
-const bool mudlet::scmIsDevelopmentVersion = !QByteArray(APP_BUILD).isEmpty();
+
+const bool mudlet::scmIsReleaseVersion = QByteArray(APP_BUILD).isEmpty();
```

Comment:
In hindsight - what happens for (Linux) packagers who - not unreasonably - take note of the advice elsewhere and apply a suffix to distinguish their own system specific version - e.g. Debian with their, typical `+deb10u1` (for products with version numbers ending in a `-1`,`-2`, etc. form) or `~deb10u1` otherwise markings? On one hand this is not a development build but it is not an official Mudlet makers produced version either? My original assumption that `APP_BUILD` would be non-blank for development builds was perhaps a bit simplistic... :man_shrugging:

Reply:
I don't know, it's not something Linux packagers do right now? I'm not sure what to answer you, the question should be directed at them.